src/init.cpp

```diff
@@ -2798,9 +2798,8 @@ bool AppInitMain(Config &config, RPCServer &rpcServer,
         return false;
     }
 
-    // Encoded addresses using cashaddr instead of base58.
-    // We do this by default to avoid confusion with BTC addresses.
-    config.SetCashAddrEncoding(args.GetBoolArg("-usecashaddr", true));
+    // Encode addresses using XAddress by default instead of CashAddr.
```

Comment:
I don't like this comment.

Should be: "By default, we use XAddress; this option will make it use the CashAddr format"